js/ui/components/clubhouse.js

```diff
@@ -75,96 +72,10 @@ function _clipToMonitor(actor) {
     actor.set_clip(0, 0, clip, actor.height);
 }
 
-function getClubhouseApp() {
-    return Shell.AppSystem.get_default().lookup_app(CLUBHOUSE_ID + '.desktop');
+function getClubhouseApp(clubhouseId = CLUBHOUSE_ID) {
```

Comment:
I don't agree with having the clubhouse id as argument, because you can pass an id that is not the Clubhouse. Maybe you can add an argument, hackVersion argument, defaulting it to 2? Or an argument `isOldClubhouse` defaulting it to false.